### DISCUSSION TITLE
enotices2 - usage of "stop-publication-esentool" Request

### DISCUSSION THREAD

**Discussion by gabrieledall** [OP-TED/eForms-SDK#291] [Q&A] [answered]
I'm not quite sure, if this is the correct platform for this question but I haven't found any other...

As far as I know: submitting eforms to TED requires the usage of the new enotices2-API.
Public API Documentation describes a stop-publication-esentool - request which is to be used in order to stop eForms - only those in status "submitted" from beeing published. 

Reading the Documentation, I understood that eForms will rest in the "submitted"-State maximum one day (from 16:01 the day being submitted to enotices2 to 16:00 the following day), because every day at 16:00 newly submitted notices will be collected and handed over to TED for being published there. eForms-status changes from "submitted" to "publishing" / "published" and a "stop-publication-esentool" cannot be exceuted having left the "submitted" status.

Is this understanding correct so far?

What will happen to those eforms that have been stoped from publication after submission? Will they be fully deleted from enotices2 or will they only change their status? 

The reason of the question is the following. National platforms are allowed to publish their eforms 2 days after submitting them to TED/notices2. In order to stick to this rules it is necessary for national platforms/esenders to understand what happened to their notices:

If a notices stays in status "submitted" for more than 2 days in the enotices2-environment and can be completely deleted after 2 days - this would probably lead to "illegal" publication states in national platforms - they could publish notices that will never be found on TED.

Would it therefore be possible to reject any "stop-publication-esentool-requests" on notices resting in "submitted" status for more than 2 days in the eNotices2-envirnoment? 

Thank you for any answer.
Gabriele Dall

**Comment by pcolast** [comment]
Dear Gabriele Dall,

eForms notices can be submitted for publication via eNotices2 application or via eNotices2 API. The usage of eNotices2 API requires manual creation of user account in eNotices2. eForms notices can be followed up through eNotices2 application or via eNotices2 API.

When submitting a notice, users can select a preferred publication date, or can choose publication “as soon as possible”. During the initial stages, OP will probably aim at publishing within five days, but it may be technically possible in the future to publish sooner provided the notice has passed automatic validation and the manual checks for lawfulness. 
The assumption "eForms will rest in the "submitted"-State maximum one day" is not correct.

Notices that have been stopped from publication after submission will remain in eNotices2 user account in status "STOPPED". They can be manually deleted through eNotices2 application.

Would it therefore be possible to reject any "stop-publication-esentool-requests" on notices resting in "submitted" status for more than 2 days in the eNotices2-envirnoment?
Currently, the system accepts "stop-publication-esentool-requests" depending on the current notice status. We will think about your question and provide the answer in the coming days.

Thank you for your question.

Best regards,
Stefan

**Comment by rouschr** [comment, marked as answer]
Hello,
note that eNotices2 is developed separately, for issues or questions concerning this project use this form:
https://enotices2.ted.europa.eu/help/contact
